src/components/Main/Navigation/NavigationContent/Users.vue

```diff
@@ -4,19 +4,40 @@
       <empty-state>Not Implemented</empty-state>
     </navigation-content-container>
     <navigation-content-container subtitle="ユーザーリスト">
-      <div
-        v-for="userList in userLists"
-        :class="$style.list"
-        :key="userList[0]"
-      >
-        <users-separator :name="userList[0]" />
+      <filter-input
+        :onSecondary="true"
+        :text="userListFilterState.query"
+        @input="setQuery"
+      ></filter-input>
```

Comment:
```suggestion
      />
```
self-closing形式で:pray:
https://jp.vuejs.org/v2/style-guide/index.html#%E8%87%AA%E5%B7%B1%E7%B5%82%E4%BA%86%E5%BD%A2%E5%BC%8F%E3%81%AE%E3%82%B3%E3%83%B3%E3%83%9D%E3%83%BC%E3%83%8D%E3%83%B3%E3%83%88-%E5%BC%B7%E3%81%8F%E6%8E%A8%E5%A5%A8